pkg/v15/resource/kubeconfig/current.go

```diff
@@ -22,7 +22,7 @@ func (r *StateGetter) GetCurrentState(ctx context.Context, obj interface{}) ([]*
 
 	r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("finding kubeconfig secret %#q", secretName))
 
-	secret, err := r.k8sClient.CoreV1().Secrets(r.resourceNamespace).Get(secretName, metav1.GetOptions{})
+	secret, err := r.k8sClient.CoreV1().Secrets(clusterGuestConfig.ID).Get(secretName, metav1.GetOptions{})
```

Comment:
```suggestion
	secret, err := r.k8sClient.CoreV1().Secrets(clusterConfig.ID).Get(secretName, metav1.GetOptions{})
```

Let's rename this to just clusterConfig. The guest cluster naming is deprecated in favor of tenant cluster. Since we need this resource long term I'd like to get the naming straight.